### PR TITLE
Created Package.nuspec file

### PR DESCRIPTION
**Pull request by arucha001**

### CODE REVIEW

File: Package.nuspec [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>TValidation</id>
```
Comment (aarbla01) [inline Package.nuspec]: As discussed, consider using a more verbose package ID, so we can clearly identify the is an ARM component package.

Comment (matdec01) [reply Package.nuspec]: Can we use namespace-like ID's? e.g. `com.arm.components.validation`?

Comment (arucha001) [reply Package.nuspec]: prefixed all id's with "ARM.Component." e.g. "ARM.Component.CFooter". Will take the prefix out while creating folder in azure blob.

Comment (arucha001) [reply Package.nuspec]: Read comment from @matdec01 after making the change, can we please quickly decide so I can proceed with finishing the Octopus setup. I am ok with any name as long as it is unique :)
The prefix can be easily changed afterwards, the prefix will be stored in Octopus which will be stripped before creating folder in Azure.

Comment (arucha001) [reply Package.nuspec]: discussed with @matdec01 and - changed the prefix in ID to "com.arm.components." & unique part is same as name in package.json.

**Pull request merged**